libhoney.go

```diff
@@ -34,15 +34,6 @@ const (
 	defaultAPIHost    = "https://api.honeycomb.io/"
 	defaultDataset    = "libhoney-go dataset"
 	version           = "1.12.0"
-
-	// DefaultMaxBatchSize how many events to collect in a batch
```

Comment:
These are all exported - we can't delete them at the moment. Instead we should add deprecation comments and delete them at a later time when we do a larger version bump.